wnprc_billing/src/org/labkey/wnprc_billing/notification/WNPRCBillingNotificationProvider.java

```diff
@@ -197,42 +197,38 @@ public void getAdditionalChargeCategoryInfo(User u, Container c, Date startDate,
 
                 totalCostPerCategory += totalCost;
             }
-            Map<String, Double> totalsMap = new HashMap<>();
-            totalsMap.put("total", (procedureRows.size()*1.00));
-            totalsMap.put("totalCost", totalCostPerCategory);
-            totalsByCategory.put(description, totalsMap);
-            ActionURL url = QueryService.get().urlFor(u, c, QueryAction.executeQuery, procedureSchema, procedureQuery);
+            Map<String, Object> categoryInfo = new HashMap<>();
+
+            // set total no. of rows per each charge category/procedure query
+            categoryInfo.put("total", (procedureRows.size() * 1.00));
```

Comment:
I missed this before. Why is it multiplying by 1.00?

Reply:
Good question, i think that the notification displays everything with 2 decimal places, even an int, not really sure why, so keeping it consistent what's in place

Reply:
This code isn't rendering to a string, so I don't see how there would be any difference between using `1.0`, `1.00`, or `1.0000000`. Something else must be responsible for the string formatting.

Reply:
I've refactored this, so that line of code is no longer valid. In fact, it should be a total quantity, and not total no. of rows in the first place, so that was my bug.